boards/milkv/duo/doc/index.rst

```diff
@@ -0,0 +1,82 @@
+.. _duo:
+
+Milk-V Duo
+##########
+
+Overview
+********
+
+See https://milkv.io/duo
+
+Supported Features
+==================
+The Milk-V Duo board configuration supports the following hardware features:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Peripheral
+     - Kconfig option
+     - Devicetree compatible
+   * - Mailbox
+     - :kconfig:option:`CONFIG_MBOX`
+     - :dtcompatible:`sophgo,cvi-mailbox`
+   * - Pin controller
+     - :kconfig:option:`CONFIG_PINCTRL`
+     - :dtcompatible:`sophgo,cvi-pinctrl`
+   * - GPIO
+     - :kconfig:option:`CONFIG_GPIO`
+     - :dtcompatible:`snps,designware-gpio`
+   * - PWM
+     - :kconfig:option:`CONFIG_PWM`
+     - :dtcompatible:`sophgo,cvi-pwm`
+   * - UART
+     - :kconfig:option:`CONFIG_SERIAL`
+     - :dtcompatible:`ns16550`
+   * - PLIC
+     - N/A
+     - :dtcompatible:`sifive,plic-1.0.0`
+   * - SysTick
+     - N/A
+     - :dtcompatible:`thead,machine-timer`
+
+Other hardware features have not been enabled yet for this board.
+
+The default configuration can be found in the defconfig file:
+
+        ``boards/milkv/duo/milkv_duo_defconfig``
+
+Programming and Debugging
+*************************
+
+Prepare a TF card and follow the instructions in the
+`official SDK <https://github.com/milkv-duo/duo-buildroot-sdk>`_ to build and
+flash the image.
+
+After that, you will get a ``fip.bin`` file at the root of the TF card. Use the
+following command to replace the RTOS image with Zephyr:
+
+.. code-block:: sh
+
+   python3 /path/to/duo-buildroot-sdk/fsbl/plat/cv180x/fiptool.py \
+       -v genfip "/path/to/target/fip.bin" \
+       --OLD_FIP="/path/to/original/fip.bin" \
+       --BLCP_2ND="build/zephyr/zephyr.bin"
```

Comment:
why can't a runner be provided? this looks hacky.

Reply:
Maybe something like `west flash -r cvi --fiptool /path/to/vendor/sdk/fiptool.py --fip-bin /path/to/sdcard/fip.bin --auto-eject`?

Reply:
yes, or rely on `--tool-opt`.

Reply:
Thank you for your suggestion. Update.